pkg/dashboard/resource/function.go

```diff
@@ -315,7 +315,6 @@ func (fr *functionResource) functionToAttributes(function platform.Function) res
 	// artifacts are created unique to the cluster not needed to be returned to any client of nuclio REST API
 	functionSpec.RunRegistry = ""
 	functionSpec.Build.Registry = ""
-	functionSpec.Build.Image = ""
```

Comment:
This field shouldn't be emptied (It is not uniquely created, and it reflects the specific name the user gave to the
 function image) we certainly want it to be returned